ISM43362/ISM43362.cpp

```diff
@@ -60,6 +61,10 @@ extern "C" int32_t ParseNumber(char *ptr, uint8_t *cnt)
         ptr++;
         i++;
     }
+    if (*ptr == 'C') {  /* input string for version is C3.x.x.x */
```

Comment:
Maybe we can add this test in the while loop below instead ?

Reply:
What if an X is 10 or more ?

Reply:
@bcostm no, as only the first character is checked. But the i++ is wrong, thx :-)

Reply:
@LMESTM we don't know if it is possible in Inventek driver...
I will keep the code liek this and add a condition to check if result is correct.